lib/sign_up.rb

```diff
@@ -0,0 +1,56 @@
+require 'bcrypt'
+require_relative './database_connection'
+
+class ChitterSignUp
```

Comment:
I feel like this is really a User class.  It seems to deal with the creation and saving of users, so I think that would be a clearer name.